### ISSUE TITLE
check-aggregate broken under sensu => 0.24.0

### ISSUE THREAD

**Issue by joaogabriel-luizalabs** [sensu-plugins/sensu-plugins-sensu#8]
Sensu has a new aggregated "method" and I think that broke check-aggregation plugin

```
root@ip-10-154-255-62:/opt/sensu/embedded/bin# ./check-aggregate.rb -c V1 -C 10 -M "oi"
Check failed to run: bad URI(is not URI?): http://localhost:4567/aggregates/V1/[:results, {:ok=>3, :warning=>0, :critical=>0, :unknown=>0, :total=>3, :stale=>0}], ["/opt/sensu/embedded/lib/ruby/2.3.0/uri/rfc3986_parser.rb:67:in `split'", "/opt/sensu/embedded/lib/ruby/2.3.0/uri/rfc3986_parser.rb:73:in `parse'", "/opt/sensu/embedded/lib/ruby/2.3.0/uri/common.rb:227:in `parse'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:276:in `parse_url'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:280:in `parse_url_with_auth'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:175:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/request.rb:41:in `execute'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/rest-client-1.8.0/lib/restclient/resource.rb:51:in `get'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-sensu-0.1.0/bin/check-aggregate.rb:95:in `api_request'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-sensu-0.1.0/bin/check-aggregate.rb:119:in `acquire_aggregate'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugins-sensu-0.1.0/bin/check-aggregate.rb:161:in `run'", "/opt/sensu/embedded/lib/ruby/gems/2.3.0/gems/sensu-plugin-1.2.0/lib/sensu-plugin/cli.rb:56:in `block in <class:CLI>'"]
```

**Comment by cwjohnston** [comment]
Hi @joaogabriel-luizalabs, thanks for reporting this. You're right that this check needs to be updated; I will take a look at it.

**Comment by RyanJarv** [comment]
+1 Just updated Sensu and running into this issue currently

**Comment by bovy89** [comment]
+1

**Comment by czervenyvlk** [comment]
I'm experiencing the same problem v0.25.1

**Comment by cwjohnston** [comment]
Closed by #12
